wherehows-dao/src/main/java/wherehows/models/view/DatasetExportPolicy.java

```diff
@@ -27,4 +27,8 @@
   private boolean containsUserActionGeneratedContent;
 
   private boolean containsUserDerivedContent;
+
+  private String modifiedBy;
+
+  private Long modifiedTime;
```

Comment:
Do we expect this field to always present? If so, can use primary type "long"

Reply:
Not aware of all the cases, so not sure if it'll always be present.